pkg/logging/logger.go

```diff
@@ -42,6 +42,21 @@ func configure(config Config) error {
 	return nil
 }
 
+// FindLogger finds a logger based on a given name
+func FindLogger(names ...string) Logger {
+	logger := root
+	for _, name := range names {
+		child, ok := logger.children[name]
+		if ok {
+			logger = child
+		} else {
+			return nil
```

Comment:
I don't necessarily think it's best to return `nil` if we can't find the logger. All loggers have a level, and if the client wants to ask about a theoretical logger that doesn't exist I think it's fine to do so and we can return the theoretical level of that theoretical logger. We should just be using `GetLogger` and returning the level of the logger the client asks for.

Reply:
Yeah. I was using Getlogger originally but I wasn't sure making this assumption about a theoretical logger is a valid assumption. I will use the GetLogger to do that but if we find it confusing then we can change it later

Reply:
Done